lib/options.json

```diff
@@ -75,14 +75,11 @@
         }
       ]
     },
-    "disableHostCheck": {
-      "type": "boolean"
-    },
-    "fs": {
+    "devMiddleware": {
```

Comment:
It is good place to do discussion, should we move all webpack-dev-middleware in the one place

/cc @hiroppy

Reply:
I think it makes sense because webpack-dev-middleware now does options validation that only allows for the properties specified. So we can just validate `devMiddleware` as an object, and if the user passes in bad webpack-dev-middleware options then webpack-dev-middleware validation will throw an error.

Reply:
I prefer this idea.

Reply:
Let's rename it to `dev`, I think it is unnecessary to have `middleware` suffix, we don't have it for other middlewares

Reply:
Also, it will be better for CLI - `webpack server --dev-something=value`